TCC.Lib/Command/CommandResult.cs

```diff
@@ -13,5 +14,30 @@ public class CommandResult
         public bool HasError => !String.IsNullOrEmpty(Errors);
         public string Command { get; set; }
         public List<string> Infos { get; set; }
+
+        public void ThrowOnError()
+        {
+            if (HasError)
+            {
+                var sb = new StringBuilder();
+                sb.AppendLine("command : " + Command);
+                sb.AppendLine("error : " + Errors);
+                throw new TccException(sb.ToString());
+            }
+        }
+    }
+
+    [Serializable]
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this implementation of 'ISerializable' to conform to the recommended serialization pattern. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3925)